<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nearest_AnalysisDialogBase</class>
 <widget class="QDialog" name="Nearest_AnalysisDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nearest_Analysis</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Application Area</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_app">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select API</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_api">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Field</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fields_list">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>161</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="log_browser">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>256</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_log_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clear log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
